examples/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "examples"
+version.workspace = true
+edition = "2021"
```

Comment:
Should be 2023 no?

Reply:
That edition = "2021" setting configures your package to be built using the Rust 2021 edition. No further configuration needed!

Refer: https://doc.rust-lang.org/edition-guide/

2021 seems to be latest edition.